### PR TITLE
Consistent help understanding updates

### LINKED ISSUE #2299: Relative order for Consistent Help (3.2.6) when serial order is relative but not visual order
The SC implies that within a page variation the visual order should be relative, but it's not required if the serial order is correct.  The note however is aimed at variations.

The note in the understanding doc states "For this SC, the same relative order can be thought of as how the content is ordered when the page is serialized. The visual position of a help mechanism is likely to be consistent across pages for the same page variation (e.g., CSS break-point). The user can cause a change, for example zoom or change orientation, which may trigger a different page variation."

I've seen instances where in the same responsive variation the help mechanism is visually in different locations but serially in the same order.  That is on one page it's at the end of content but before the footer in the main area and on another page it's also between the content and the footer but on the right side (same variation/viewport).  Does that meet the SC as the note implies?  It seems like it would be best otherwise - but would pass.

### LINKED ISSUE #2414: Benefit text + Examples Understanding 3.2.6 Consistent Help not the right fit anymore?!
In the Understanding doc there is explained what kind of help is good/better/best
https://w3c.github.io/wcag/understanding/consistent-help.html#benefits

This was once applicable when the SC was in its early stage.
As it now only is all about the consistent location, the text should be adjusted to match that situation.

### PR DESCRIPTION
**Pull request by alastc**
Closes #2414

Quite an overhaul, I created a section called "Support for people with cognitive and learning disabilities" at the bottom of the intent, and put the more 'advice' type content there, with a bit at the top saying "not required by the SC".

Also revised the examples and benefits.

Closes #2299 
(Line 53 added)

### CODE REVIEW

**Comment by bruce-usab** [comment]
ping to @melaniephilipp

**Comment by bruce-usab** [comment]
closes https://github.com/w3c/wcag/issues/2414

**Comment by bruce-usab** [comment]
@michael-n-cooper please enable @melaniephilipp for reviewer and assignee

File: understanding/22/consistent-help.html [hunk 1]
```diff
@@ -36,86 +36,95 @@ <h2>Consistent Help</h2>
...
-				<li>Content is archived, or </li>
-            <li>When finding help would invalidate the activity, such as in a testing situation. </li>
-         </ul>
+         <p>Without help, some users may abandon the task. They may also fail to correctly complete a task, or they may require assistance from people who do not necessarily keep private information secure.</p>
 
-        <p>It is also not the intent of this Success Criterion to require a human be available at all times. If the human contact is not available during certain hours or certain days, information should be provided so the user can tell when it will be available. </p>
-   </section>
+         <p>It is not the intent of this Success Criterion to require authors to provide help / access to help. The Criterion only requires that <em>when</em> one of the listed forms of help is available across multiple pages that it be in a consistent location. It does not require authors to provide help information on PDFs or other static documents that may be available for viewing/download from the Web pages, these are not considered part of the "set of web pages" from which they are downloaded.</p>
```
Comment (dbjorge) [inline understanding/22/consistent-help.html:62]: ed: the last sentence in this paragraph is a run-on. Consider replacing the comma between "pages" and "these" with either a sentence break or a semicolon.

File: understanding/22/consistent-help.html [hunk 2]
```diff
@@ -36,86 +36,95 @@ <h2>Consistent Help</h2>
...
 
-        <p>The human contact details enable users to connect with the  organization or the part of the organization that can assist with the content. For example, an online jobs / recruitment portal may provide a contact method for the team that supports the recruitment portal and not a catch-all for the entire company. Each layer of contact added prolongs the time before the user will receive help.</p>
+            <p>For pages for which no human support is available it helps if a self-help option says that no human support is available. Self-help options can go beyond allowing the user to search within the site. Contextual help is still recommended (see <a href="help">Success Criterion 3.3.5</a> for more information) but a self-help option provides a single location that makes it easier for people with cognitive disabilities to understand what help is available without having to hunt for it. While some people may easily be able to identify that no support would be available for a particular type of Web site, this may not be apparent to some users with disabilities.</p>
```
Comment (dbjorge) [inline understanding/22/consistent-help.html:88]: ed: in third sentence after parenthetical, there should be a comma before "but"

File: understanding/22/consistent-help.html [hunk 3]
```diff
@@ -36,86 +36,95 @@ <h2>Consistent Help</h2>
...
+            </ul>
 
-        <p>When a user is quickly able to find help, they are able to complete the task even if they encounter challenges.</p>
+            <p>The order of the types of help listed in the Success Criterion does not imply priority.</p>
+         </section>
+         <section id="support-for-coga">
+            <h3>Support for people with cognitive and learning disabilities</h3>
 
-         <h2>Help Mechanisms</h2>
+            <p>This section is not required by the Consistent Help success criterion, but provides advice related to <a href="https://www.w3.org/TR/coga-usable/#support-user-story">Providing Help and Support from COGA Usable</a>.</p>
```
Comment (dbjorge) [inline understanding/22/consistent-help.html:82]: "COGA Usable" is the term used in the URL fragment, but isn't used by the linked document to describe its own title. Would be more consistent to use the title of the document as it appears in the document to label this link? ("Making Content Usable for People with Cognitive and Learning Disabilities")

File: understanding/22/consistent-help.html [hunk 4]
```diff
@@ -36,86 +36,95 @@ <h2>Consistent Help</h2>
...
+            <ul>
+               <li>recognize misspelled words,</li>
+               <li>provide human contact details if the chatbot is unable to provide a satisfactory response after 3 attempts, and</li>
+               <li>can be dismissed with a single interaction, and recalled using a link or button.</li>
+            </ul>
 
-        <p>Chatbots can work more effectively for everyone, and particularly for people with cognitive disabilities if they:</p>
+            <p>This criterion does not require that a site provide a help mechanism. However, when help is available:</p>
+            <ul>
+               <li>people who may have difficulty locating help are more likely to find it and complete their task.</li>
```
Comment (dbjorge) [inline understanding/22/consistent-help.html:100]: ed: inconsistent capitalization (this and the following line should probably be capitalized as L102 is)

File: understanding/22/consistent-help.html [hunk 5]
```diff
@@ -36,86 +36,95 @@ <h2>Consistent Help</h2>
...
+               <li>users that experience cognitive fatigue or cognitive shut down will be able to reserve their energy for the task, instead of using it to find support.</li>
+               <li>Enabling users (especially those with cognitive disabilities) to find solutions while expressing their question using their own words (for example by interacting with a <q>chatbot</q>) increases their chances of success for completing a task.</li>
```
Comment (dbjorge) [inline understanding/22/consistent-help.html:102]: ed: "chatbot" is enclosed in `<q>` here but not on L94 - should be consistent

**Comment by mbgower** [comment]
Nice work. I recommend adopting all @dbjorge's suggestions. As well, I believe it is recommended by COGA to replace "e.g." with "for example".

**Comment by alastc** [comment]
Approved: https://www.w3.org/2023/02/14-ag-minutes#item09

**Pull request merged**